### PR TITLE
navigator bar (close #2)

### LINKED ISSUE #2: feat: main header pages
Make the main header with `nav` tag and add link to other pages. Be aware that the header must be added in all pages.

### PR DESCRIPTION
**Pull request by Weslley-Mistura**
Create the navigator bar in the 3 pages (home, about and contact)

### CODE REVIEW

File: styles/about.css [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+
+.menu-header ul {
+  list-style: none;
+}
+
+.menu-header ul li {
+  display: inline;
+}
+
+ul {
```
Comment (lucas-segundo) [inline styles/about.css]: @Weslley-Mistura , avoid applying global styles for a tag. I suggest you to do something as the line 46 of the code.

File: styles/about.css [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+  margin-right: 80px;
+}
+
+.about {
+  margin-right: 80px;
+}
+
+.contact {
+  margin-right: 70px;
+}
```
Comment (lucas-segundo) [inline styles/about.css]: These styles seem too familiar, how about we simplified it:

`.navigator li {
  margin-right: 80px;
}`

**Review by lucas-segundo — CHANGES_REQUESTED** [review]
Check my suggestion, but you have done a great work!

File: styles/about.css (hunk 2 above)
Comment (Weslley-Mistura) [reply styles/about.css]: You just to simplify by reducing the number of lines?

Comment (lucas-segundo) [reply styles/about.css]: Yep. If you replace .home, .about and .contact to my code, It will work like the same way. Try it. The code will be simpler.

**Review by lucas-segundo — APPROVED** [review]
All good!

File: styles/about.css (hunk 2 above)
Comment (lucas-segundo) [reply styles/about.css]: Ok! You fixed with the last commit.

**Pull request merged**